server/bin/pbench-audit-server.sh

```diff
@@ -727,26 +730,22 @@ if [[ $? -ne 0 ]]; then
 fi
 eTS=$(timestamp)
 if [[ -s ${archive_report} ]]; then
-    log_info "" "${report}"
-    log_info "start-${sTS}: archive hierarchy: $ARCHIVE" "${report}"
+    log_info "${SingleNewline}start-${sTS}: archive hierarchy: $ARCHIVE" "${report}"
```

Comment:
That's better, but I'd still like to see if we can leave the strings alone and still  have everything work.

Reply:
Using %b instead of %s should solve this problem I think.

Reply:
ok